src/base-element.js

```diff
@@ -950,4 +950,8 @@ export class BaseElement {
    * @public
    */
   onLayoutMeasure() {}
+
+  user() {
```

Comment:
@lannka actually how do I get (this) for some other files that don't directly have access to BaseElement? I feel like i'm running into the same problem with user(this.element).